src/components/ContentSidebar/ActivitySidebar.js

```diff
@@ -405,7 +421,9 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
                 },
             },
         });
-        this.errorCallback(e);
+
+        const error = createErrorFromResponse(e);
+        this.errorCallback(error, ERROR_CODE_FETCH_CURRENT_USER);
```

Comment:
It seems like error codes should be defined in the API layer rather than in the containers. Otherwise any consuming component would need to duplicate this logic.

Reply:
Agreed